mrp_bom_version/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# (c) 2015 Alfredo de la Fuente - AvanzOSC
+# (c) 2015 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "MRP - BoM version",
+    "summary": "BoM versioning",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "OdooMRP team,"
+    "AvanzOSC,"
+    "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+    "Odoo Community Association (OCA)",
+    "website": "http://www.odoomrp.com",
```

Comment:
the site doesn't seem to exist anymore could put the link to the oca repo instead ?

Reply:
done :)